src/components/Text/Text.story.js

```diff
@@ -1,42 +1,33 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withInfo } from '@storybook/addon-info';
+import { select, boolean } from '@storybook/addon-knobs';
 import { GROUPS } from '../../../.storybook/hierarchySeparators';
 
 import withTests from '../../util/withTests';
 import Text from '.';
 
+const elements = ['p', 'article', 'div'];
```

Comment:
How about adding `<span>`? Did you exclude it for a particular reason?

Reply:
In theory, this can be any element. I just picked the ones that had an individual story before.

Reply:
I've added more element options. 👍